packages/ui/src/apis/register.ts

```diff
@@ -14,3 +14,53 @@ export const register = async (username: string, password: string) => {
     })
     .json<Response>();
 };
+
+export const sendPhonePasscode = async (phone: string) => {
+  return ky
+    .post('/session/register/passwordless/phone/send-passcode', {
+      json: {
+        phone,
+      },
+    })
+    .json();
+};
```

Comment:
Can we change the folder path from `apis` to `services`, as they are actually front-end service calls to call the backend APIs

Reply:
That gonna be a cross ui project-wised update, prefer to update in a separate pr. 
Besides @logto-io/eng for alignment of the naming conversion, `api` or `services`.

Reply:
Both are OK to me.

Reply:
https://linear.app/silverhand/issue/LOG-2054/api-folder-name-convention